esx_service/vmdk_ops.py

```diff
@@ -288,7 +288,7 @@ def cloneVMDK(vm_name, vmdk_path, opts={}, vm_uuid=None, datastore_url=None):
     lockname = "{}.{}.{}".format(src_datastore, tenant_name, src_volume)
     with lockManager.get_lock(lockname):
         # Verify if the source volume is in use.
-        attached, uuid, attach_as = getStatusAttached(src_vmdk_path)
+        attached, uuid, attach_as, _ = getStatusAttached(src_vmdk_path)
         if attached:
             if handle_stale_attach(vmdk_path, uuid):
                 return err("Source volume cannot be in use when cloning")
```

Comment:
How about changing this error message to:
"Source volume {0} in use by VM {1} and can't be cloned.".format(vol_name, attached_vm_name)